server/src/repositories/userRepository.ts

```diff
@@ -169,6 +177,50 @@ const getUserDataByUserId = async (userId: string): Promise<UserData | undefined
 	return userDataMapper(res.rows[0]);
 };
 
+const getCompletenessByUserId = async (userId: string): Promise<UserCompletness | undefined> => {
+	const query = {
+		text: 'select is_complete from users where id = $1',
+		values: [userId]
+	};
+	const res = await pool.query(query);
+	if (!res.rowCount) {
+		return undefined;
+	}
+	return userCompletnessMapper(res.rows[0]);
+};
+
+const updateCompletenessByUserId = async (userId: string, value: boolean): Promise<void> => {
+	const query = {
+		text: 'update users set is_complete = $2 where id = $1',
+		values: [userId, value]
+	};
+	await pool.query(query);
+};
+
+const userDataIsNotNULL = async (userId: string): Promise<boolean> => {
+	const query = {
+		text: 'select * from users where id = $1 and gender is not null',
```

Comment:
Noticed only gender field has been checked in query. Is there some reason for that? Shouldn't we check all the fields?